cli/lsp/documents.rs

```diff
@@ -1140,8 +1144,9 @@ impl Documents {
 
   /// Return a document for the specifier.
   pub fn get(&self, original_specifier: &ModuleSpecifier) -> Option<Document> {
+    let mark = self.performance.mark("lsp.get_document");
```

Comment:
I feel like this is always going to be extremely fast. I'm not sure this is worth measuring this way. Has someone taken a flame graph and found this part to be slow?